TS2309: An export assignment cannot be used in a module with other exported elements

This module causes ts builds to fail with TS2309

package.json

```json
{
  "private": true,
  "dependencies": {
    "moving-average": "^1.0.1",
    "typescript": "^4.1.3"
  }
}
```

tsconfig.json

```json
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "moduleResolution": "node",
    "strict": true,
    "target": "ES2019"
  },
  "include": [
    "script.ts"
  ],
  "exclude": [
    "**/node_modules"
  ]
}
```

script.ts

```ts
import MovingAverage from 'moving-average'
```

compile with

```sh
./node_modules/.bin/tsc
```

error:

```
node_modules/moving-average/index.d.ts:12:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

12 export = MovingAverage
   ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,12 +1,15 @@
+declare namespace MovingAverage {
+  export interface IMovingAverage {
+    variance: () => number
+    movingAverage: () => number
 
-export function MovingAverage (timespan:number): MovingAverage
-export interface MovingAverage {
-  variance: () => number
-  movingAverage: () => number
+    deviation: () => number
+    forecast: () => number
 
-  deviation: () => number
-  forecast: () => number
-
-  push: (time: number, value: number) => void
+    push: (time: number, value: number) => void
+  }
 }
+
+declare function MovingAverage(timespan: number): MovingAverage.IMovingAverage
+
 export = MovingAverage

